src/main/res/values/strings.xml

```diff
@@ -139,23 +139,23 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about newer applications</string>
```

Comment:
```suggestion
  <string name="update_notifications">Notifications about newer application versions</string>
```
%d applications have newer versions available.

Reply:
![device-2020-07-06-110636](https://user-images.githubusercontent.com/24494863/86570628-bf452700-bf5f-11ea-87da-53538296fc1c.png)

"Notifications about newer applications" instead of "Update notifications" is too long. And now you want "Notifications about newer application versions" which is even longer :/